.travis.yml

```diff
@@ -40,9 +47,12 @@ before_cache:
   - rm -rf ${HOME}/.cache/pants/lmdb_store
   # Render a summary of what is left in the home directory, to assist with further pruning of
   # the cache.
-  - du -d2 ${HOME} | sort -r -n
+  - du -m -d2 ${HOME} | sort -r -n
 
 cache:
+  # The default timeout is 180 seconds, and our larger cache uploads exceed this.
+  # TODO: Figure out why we have such large caches (2-7GB) and try to trim them.
+  timeout: 500
```

Comment:
Some of this should be ameliorated by this PR itself, since rust will only be bootstrapped on one shard.

Reply:
I hope so, but I needed this just to get things to pass...